packages/solid-crs-webid/data/local/.acl

```diff
@@ -0,0 +1,27 @@
+@prefix acl: <http://www.w3.org/ns/auth/acl#>.
+@prefix foaf: <http://xmlns.com/foaf/0.1/>.
+
+<#authorization>
+    a               acl:Authorization;
+    acl:agentClass  foaf:Agent;
+    acl:mode        acl:Read;
+    acl:mode        acl:Write;
+    acl:mode        acl:Append;
+    acl:mode        acl:Delete;
+    acl:mode        acl:Control;
+    acl:accessTo    <./>;
+    acl:default     <./>.
+
+<#minter-append-webids>
+    a acl:Authorization;
+    acl:agent <http://localhost:3002/dgt-id-proxy/card#me>;
+    acl:accessTo <./>;
+    acl:mode
+        acl:Read, acl:Append.
+
+<#minter-control-webids>
+    a acl:Authorization;
+    acl:agent <http://localhost:3002/dgt-id-proxy/card#me>;
+    acl:default <./>;
+    acl:mode
+        acl:Read, acl:Write, acl:Control.
```

Comment:
@AbelVandenBriel what should these agents be set to?

Reply:
(localhost:3002 is the manage application)

Reply:
We will have to:

- add an .acl file with this information in it to the root of webid.netwerkdigitaalerfgoed.nl. The `acl:agent` should be replaced with https://webid.netwerkdigitaalerfgoed.nl/dgt-id-proxy/card#me
- add a webid document to dgt-id-proxy. Basically just registering above agent.